wayshot/src/wayshot.rs

```diff
@@ -33,29 +31,75 @@ where
 }
 
 fn main() -> Result<()> {
+    // cli args
     let cli = cli::Cli::parse();
+
+    // config path
+    let config_path = dirs::config_local_dir()
```

Comment:
Nit: extract to separate function of the sort -> `fn get_config_path(...)`

Reply:
Done (I think).